kolibri/core/package.json

```diff
@@ -21,7 +21,7 @@
     "js-cookie": "^3.0.5",
     "knuth-shuffle-seeded": "^1.0.6",
     "kolibri-constants": "0.2.5",
-    "kolibri-design-system": "4.2.0",
+    "kolibri-design-system": "https://github.com/AlexVelezLl/kolibri-design-system.git#110b3ecb909aed2015b3cfdf66ed501fb442447e",
```

Comment:
Note that we can merge this PR after a new KDS release is out and this reference will need to be changed back to the npm package before merging. Thank you for preparing it for testing! Just leaving the note here so we don't forget when the time comes.